jni_gen/bin/build_summarizer.dart

```diff
@@ -0,0 +1,53 @@
+// this script takes a vendored ApiSummarizer and builds with maven.
```

Comment:
should this be `bin/setup.dart` instead?

Reply:
Is `setup.dart` a common convention? If so I will rename it.

Reply:
More common than a specific name. The `bin/*.dart` files are not easily discoverable. `dart run jni_gen:setup` seems clearer in docs to me than `dart run jni_gen:build_summarizer`. The users of `package:jni_gen` do not have to know that we have two separate components on the inside (the summarizer and generator). They just want to run one setup step and be done.